src/sage/rings/polynomial/multi_polynomial_libsingular.pyx

```diff
@@ -5757,6 +5757,83 @@ cdef class MPolynomial_libsingular(MPolynomial_libsingular_base):
         else:
             return self * self.denominator()
 
+    def in_subalgebra(self, J, algorithm="algebra_containment"):
+        """
+        Return whether this polynomial is contained in the subalgebra
+        generated by ``J``
+
+        INPUT:
+
+        - ``J`` -- list of elements of the parent polynomial ring
+
+        - ``algorithm`` -- can be ``None`` (the default), "algebra_containment",
+          "inSubring", or "groebner".
+
```

Comment:
Now that you removed `None` from the code below, this docstring should be updated, like
```
        - ``algorithm`` -- one of  "algebra_containment" (default),
           "inSubring", or "groebner".
```

Reply:
Or see the comment above.

Reply:
> Now that you removed `None` from the code below, this docstring should be updated, like
> 
> ```
>         - ``algorithm`` -- one of  "algebra_containment" (default),
>            "inSubring", or "groebner".
> ```

Fixed.